crates/protogen/proto/rpcsrv/common.proto

```diff
@@ -0,0 +1,31 @@
+// Types shared between our "external" and "internal" apis.
+//
+// See
+// <https://www.notion.so/glaredb/Distributed-Exec-f1e40e1dac174762b4f0fe4f6ac03ac9?pvs=4#c986d64150ac4b079d431d3d36f25f0f>
+
+syntax = "proto3";
+
+package rpcsrv.common;
+
+/// A single batch as the result of query execution.
+message ExecutionResultBatch {
+  // Session id.
+  //
+  // TODO: This will be changed to database_id when we remove the 1:1
+  // relationship between local and remote sessions.
```

Comment:
should we change or add a new number for this one for deprecation purposes?

Reply:
I'm inclined to say no since to the client, whether we use a session id or database id here doesn't matter as it's just some identifier for the remote session. This (and `work_id`) are to identify the session+stream this batch is for.